examples/ExchangeOracles/contracts/ExchangeMarket.aes

```diff
@@ -0,0 +1,58 @@
+contract ExchangeMarket =
+
+  record state = {
+    aePrice : int,
+    tokenPrice: int}
+
+  type query_type = string
+  type answer_type = int
+  type oracle_id = oracle(query_type, answer_type)
+  type query_id = oracle_query(query_type, answer_type)
+
+  public stateful function init(_aePrice: int, _tokenPrice: int) = 
```

Comment:
sophia style usually doesn't use the _ but rather suffix `'` if it is exactly the same variable as used later as none _